### ISSUE TITLE
Frames must have accesible names

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1123]
Ensures <iframe> and <frame> elements have an accessible name.

To solve this problem, you need to fix at least (1) of the following:

- [ ] Element has no title attribute
- [x] aria-label attribute does not exist or is empty
- [ ] aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- [ ] Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by davidbenque** [comment]
as mentioned in #1169 the title of the page/iframe should be in the format "CryptPad - " + app name, making use of localized strings

**Comment by dariiing** [comment]
Opening this issue again because of recent tests on `v2024.3.0`
